six/modules/c++/six.sidd/include/six/sidd/SFA.h

```diff
@@ -68,16 +70,18 @@ struct SFAPoint : public SFAGeometry
     double x, y, z, m;
 
     SFAPoint() :
-        SFAGeometry(TYPE_NAME)
+        SFAGeometry(TYPE_NAME), x(0), y(0),
+        z(::six::Init::undefined<double>()),
+        m(::six::Init::undefined<double>())
     {
-        x = 0;
-        y = 0;
         z = ::six::Init::undefined<double>();
```

Comment:
z and m can be removed from the constructor body.  Also, put x and y on their own lines